tests/designday_test.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+
+import unittest
+import os
+from ladybug.location import Location
+from ladybug.analysisperiod import AnalysisPeriod
+from ladybug.designday import \
+    Ddy, \
+    DesignDay
+
+
+class DdyTestCase(unittest.TestCase):
+    """Test for (ladybug/designday.py)"""
+
+    # preparing to test.
+    def setUp(self):
+        """set up."""
+        pass
+
+    def tearDown(self):
+        """Nothing to tear down as nothing gets written to file."""
+        pass
+
+    def test_import_ddy(self):
+        """Test import standard ddy."""
+        relative_path = './tests/ddy/chicago.ddy'
+        abs_path = os.path.abspath(relative_path)
+
+        ddy_rel = Ddy.from_ddy_file(relative_path)
+        ddy = Ddy.from_ddy_file(abs_path)
+
+        # Test imports don't break
+        assert ddy.file_path == abs_path
+        assert ddy_rel.file_path == os.path.normpath(relative_path)
+        assert ddy.file_name == 'chicago.ddy'
+
+    def test_ddy_from_design_day(self):
+        """Test ddy from design day method."""
+        relative_path = './tests/ddy/chicago_monthly.ddy'
+        ddy = Ddy.from_ddy_file(relative_path)
+        new_ddy = Ddy.from_design_day(ddy.design_days[0])
+        assert ddy.location == new_ddy.location
+        assert ddy.design_days[0] == new_ddy.design_days[0]
+
+    def test_standard_ddy_properties(self):
+        """Test properties of a standard ddy."""
+        relative_path = './tests/ddy/tokyo.ddy'
+
+        ddy = Ddy.from_ddy_file(relative_path)
+
+        # Test accuracy of import
+        assert ddy.location.city == 'TOKYO HYAKURI_JPN Design_Conditions'
+        assert -0.1 < ddy.location.latitude - 36.18 < 0.1
```

Comment:
You might want consider `pytets.approx` in such cases: https://docs.pytest.org/en/latest/reference.html#pytest-approx
`assert ddy.location.latitude - 36.18 = pytets.approx(0.1)`

Reply:
Ok, I've updated all test like this to use pytest